src/lib/objects.js

```diff
@@ -0,0 +1,74 @@
+const createPerson = (name, age) => {
+  const person = {
+    name,
+    age
+  };
+  return person;
+};
+
+const getName = object => {
+  return object.name;
+};
+
+const getProperty = (property, object) => {
+  return object[property];
+};
+
+const hasProperty = (property, object) => {
+  return object.hasOwnProperty(property);
```

Comment:
You can fix the eslint error being thrown here by using the array style notation for accessing the properties of the object as follows:
```
  return object[property];
```

This gets rid of the error, but it doesnt return the value we want.

When you are looking for return a boolean as you are here, we could use a ternary:
```
  return object[property] ? true : false;
```
While this works, it could be reduced further by using a double negation:
```
  return !!object[property];
```
Much cleaner :)